### ISSUE TITLE
UI - Failed Unit Test, Locally

### ISSUE THREAD

**Issue by jkmarx** [parklab/refinery-platform#1427]
* Specific code commit: 601615c284b46499099b9edfec0c1d5328144833
* Version of the web browser and OS: Chrome 53.0.2785.89
* Environment where the error occurred (Vagrant VM and site conf mode or AWS instance): 
Dev mode

### Steps to reproduce
Please list all the actions and the input data used:
1.  Switch to develop
2. Run grunt test

### Observed behavior
Travis builds are passing.
Locally:
``` 
Running "karma:unit" (karma) task
26 09 2016 10:02:19.582:INFO [karma]: Karma v0.13.22 server started at http://localhost:9999/
26 09 2016 10:02:19.606:INFO [launcher]: Starting browser PhantomJS
26 09 2016 10:02:20.567:INFO [PhantomJS 2.1.1 (Linux 0.0.0)]: Connected on socket /#qTTkRYVpnAKQvlHJAAAA with id 12769061
PhantomJS 2.1.1 (Linux 0.0.0): Executed 136 of 460 SUCCESS (0 secs / 0.579 secs)
PhantomJS 2.1.1 (Linux 0.0.0) DataSetImport.directive.isaTabImport: unit tests DOM should replace custom element with template FAILED
        Expected 'dataUploadTutorialStep6' to be 'isa-tab-import-form'.
        /vagrant/refinery/ui/source/js/data-set-import/directives/isa-tab-import.spec.js:37:42
PhantomJS 2.1.1 (Linux 0.0.0): Executed 460 of 460 (1 FAILED) (2.574 secs / 1.923 secs)
Warning: Task "karma:unit" failed. Use --force to continue.
Aborted due to warnings.
```
### Expected behavior
When local grunt tests fail, travis builds should fail.

**Comment by mccalluc** [comment]
When adding the selenium tests, I changed `grunt` to `grunt make`: I don't know why. @scottx611x has fixed the the UI test, and in the same PR is fixing travis.

**Comment by scottx611x** [comment]
This has been addressed as of commit: 77c595b0b004c49a8b70ca5cf67355efe0917a7b
